examples/js/e-commerce-umd/index.html

```diff
@@ -0,0 +1,125 @@
+            />
+          </svg>
+        </a>
+      </p>
+
+      <p class="header-title">Stop looking for an item — find it.</p>
+
+      <div data-widget="searchbox"></div>
+    </header>
+
+    <main class="container">
+      <div class="container-wrapper">
+        <section class="container-filters">
+          <div class="container-header">
+            <h2>Filters</h2>
+
+            <div data-widget="clear-filters" data-layout="desktop"></div>
+            <div data-widget="results-number-mobile" data-layout="mobile"></div>
+          </div>
+
+          <div class="container-body">
+            <div data-widget="categories"></div>
+            <div data-widget="brands"></div>
+            <div data-widget="price-range"></div>
+            <div data-widget="free-shipping"></div>
+            <div data-widget="ratings"></div>
+          </div>
+        </section>
+
+        <footer class="container-filters-footer" data-layout="mobile">
+          <div
+            class="container-filters-footer-button-wrapper"
+            data-widget="clear-filters-mobile"
+          ></div>
+
+          <div
+            class="container-filters-footer-button-wrapper"
+            data-widget="save-filters-mobile"
+            data-action="close-overlay"
+          ></div>
+        </footer>
+      </div>
+
+      <section class="container-results">
+        <header class="container-header container-options">
+          <div class="container-option" data-widget="sort-by"></div>
+          <div class="container-option" data-widget="hits-per-page"></div>
+        </header>
+
+        <div data-widget="hits"></div>
+        <div class="hits-empty-state">
+          <div
+            data-widget="clear-filters-empty-state"
+            class="hits-empty-results"
+          ></div>
+        </div>
+
+        <footer class="container-footer">
+          <nav data-widget="pagination"></nav>
+        </footer>
+      </section>
+    </main>
+
+    <aside data-layout="mobile">
+      <button class="filters-button" data-action="open-overlay">
+        <svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 16 14">
+          <path
+            d="M15 1H1l5.6 6.3v4.37L9.4 13V7.3z"
+            stroke="#fff"
+            stroke-width="1.29"
+            fill="none"
+            fill-rule="evenodd"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          />
+        </svg>
+
+        Filters
+      </button>
+    </aside>
+
+    <script src="/packages/instantsearch.js/dist/instantsearch.production.min.js"></script>
```

Comment:
what is this URL relative to?

Reply:
To the root of the monorepo: https://parceljs.org/features/dependency-resolution/#absolute-specifiers.